code/game/objects/landmarks/general.dm

```diff
@@ -0,0 +1,17 @@
+/obj/landmark/gun_paper_gen
```

Comment:
Give it a better and understandable by mapper who has barely any idea on how serials work name

Reply:
renamed it to IH_Armoury_Gun_Serial_Printer